advanced_source/mobile_android.rst

```diff
@@ -0,0 +1,130 @@
+Image Classification from an Android App
+========================================
+
+Starting with PyTorch 1.3, you will be able to use PyTorch models that have
+been serialized via TorchScript directly from within Android apps. This
+tutorial will provide a simple example of:
+
+* Loading a serialized PyTorch model into an Android app from Java
+* Use the new PyTorch Java API to define a ``Tensor`` to be fed through the
+  model
+
+Note: this is essentially a walkthrough of the "highlights" from the ``vision``
+portion of the example Android app we provided with PyTorch 1.3.
+
+* The GitHub repo for the full app is `here <https://github.com/zalandoresearch/fashion-mnist>`__.
+* The "vision" files are `here <https://github.com/pytorch/android-demo-app/tree/ik_android_demo_app_init/PyTorchDemoApp/app/src/main/java/org/pytorch/demo/vision>`__.
+
+Setting the scene
+~~~~~~~~~~~~~~~~~
+
+A common use case for mobile apps, as opposed to training models from scratch on the device, is
+loading a serialized pre-trained model and then passing data through it. This tutorial will walk
+through this scenario; suppose you have serialized a model using the following code:
+
+.. code:: python
+
+    import torch
+    import torchvision
+
+    model = torchvision.models.resnet18(pretrained=True), torch.rand(1, 3, 224, 224)
+    model.eval()
+    torch.jit.trace(model, input).save("resnet18.pt")
+
+You can use PyTorch in your project by adding a gradle dependency to the latest snapshot in the
+sonatype repository.
+
+.. code:: java
+
+    repositories {
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots"
+        }
+    }
+
+    dependencies {
+        implementation 'org.pytorch:pytorch_android:0.0.8-SNAPSHOT'
+    }
```

Comment:
nit:
We have 2 places where we publish pytorch_android:
1) every night on our CI we publish to  "https://oss.sonatype.org/content/repositories/snapshots" - nexus sonatype repository and we publish 'snapshot' (synonym to 'nightly' for maven artifacts :)) version ends with '-SNAPSHOT'
2) Manual publishing goes to bintray and jcenter repo, that is kind of 'release'

snapshots dependencies may feel for users less reliable  than 'release', we already have 'release' jcenter() versions published.

I think it will be better to use 'release' versions in tutorials, but also mention somewhere that we also have snapshots which diverse with master not more than 24 hours :)

For release deps it will look:

```
repositories {
    jcenter()
}

dependencies {
    implementation 'org.pytorch:pytorch_android:0.0.8'
    implementation 'org.pytorch:pytorch_android_torchvision:0.0.8'
}
```

Reply:
Hey @IvanKobzarev, thanks for this!

FYI: after yesterday's conversation, we are _not_ going to do mobile tutorials on pytorch.org/tutorials; we're just going to have even simpler "Hello, World!" examples that we host in the appropriate places on pytorch.org/mobile. Make sense?

So, please just make sure these changes are reflected in the Google Doc; that doc is what we'll use to populate the website. 

I'll actually close this PR now.